channels/api.py

```diff
@@ -209,11 +211,16 @@ def _get_client(user):
     Get a configured Reddit client_id
 
     Args:
-        user (User): the authenticated user
+        user (User): the authenticated user, or None for anonymous user
 
     Returns:
         praw.Reddit: configured reddit client
     """
+    if user.is_anonymous:
+        return praw.Reddit(
```

Comment:
One thing we'll need to keep an eye out for here is that it doesn't take advantage of the cached tokens so we should keep an eye on reddit load/latency when this release goes out. I don't see a huge need to address this until it becomes a problem though.

Reply:
:+1: